python_modules/dagster-graphql/dagster_graphql/implementation/run_config_schema.py

```diff
@@ -18,25 +20,20 @@
 
 @capture_error
 def resolve_run_config_schema_or_error(
-    graphene_info: ResolveInfo, selector: PipelineSelector, mode: Optional[str]
+    graphene_info: ResolveInfo, selector: PipelineSelector, mode: Optional[str] = None
 ) -> "GrapheneRunConfigSchema":
-    from ..schema.errors import GrapheneModeNotFoundError
     from ..schema.run_config import GrapheneRunConfigSchema
 
     check.inst_param(selector, "selector", PipelineSelector)
-    check.opt_str_param(mode, "mode")
 
-    external_pipeline = get_external_pipeline_or_raise(graphene_info, selector)
-
-    if mode is None:
-        mode = external_pipeline.get_default_mode_name()
+    if mode and mode != DEFAULT_MODE_NAME:
+        return GrapheneModeNotFoundError(selector=selector, mode=mode)
```

Comment:
shouldn't be possible anymore right?

Reply:
This is just to keep the behavior of the GQL API constant. Added comment.